packages/enzyme/src/ShallowWrapper.js

```diff
@@ -197,6 +251,8 @@ class ShallowWrapper {
         throw new TypeError('ShallowWrapper can only wrap valid elements');
       }
 
+      const getChildContextSpy = lifecycles.getChildContext.calledByRenderer
+        ? spyOnGetChildContextInitialRender(nodes, adapter) : null;
```

Comment:
if the entire ternary can't fit on one line, then the `?` and `:` need to both be on their own lines